R/pkg/R/functions.R

```diff
@@ -2452,11 +2453,18 @@ setMethod("date_format", signature(y = "Column", x = "character"),
 #'}
 #' @note from_json since 2.2.0
 setMethod("from_json", signature(x = "Column", schema = "structType"),
-          function(x, schema, ...) {
+          function(x, schema, asJsonArray = FALSE, ...) {
```

Comment:
@felixcheung, I am still not fully sure of this name, `asJsonArray`. I am okay if you have a better one.